erna/scripts/submit_runlist.py

```diff
@@ -0,0 +1,65 @@
+import click
+import pandas as pd
+
+from ..automatic_processing.database_utils import insert_new_jobs
+from ..utils import load_config
+from datetime import date
+
+from ..automatic_processing.database import (
+    database,
+    RawDataFile,
+    Jar,
+    XML,
+    Queue,
+    Job,
+)
+
+
+@click.command()
+@click.argument('runlist')
+@click.argument('jar')
+@click.argument('xml')
+@click.option(
+    '-p', '--priority', default=5, type=int,
+    help='Priority of the jobs, lower value means more important'
+)
+@click.option(
+    '-q', '--queue', default='fact_short',
+    help='Name of the queue to use'
+)
+@click.option('--config', '-c', help='Path to the yaml config file')
+def main(runlist, jar, xml, priority, queue, config):
+    '''
+    Submit automatic processing jobs for a given runlist
+
+    Arguments
+
+    RUNLIST: csv file with columns `night, run_id`
+    JAR: version of the fact-tools jar
+    XML: Name of the xml file to use
+
+    Jar and XML must be uploaded to the processing db using erna_upload
+    '''
+    config = load_config(config)
+
+    database.init(**config['processing_database'])
+
+    jar = Jar.select(Jar.id, Jar.version).where(Jar.version == jar).get()
+    xml = XML.get(name=xml, jar=jar)
+    queue = Queue.get(name=queue)
+
+    runs = pd.read_csv(runlist)
```

Comment:
I don't know what csv files your adding but this call expects 'pandas style' csv files. Meaning it takes the first column as the index, if this is intended it should be explained in the docsting, otherwise fix this call to not do that (`index_col=False`).

Reply:
This is not true. The only requirement is to have the column names in the first row.